src/App.js

```diff
@@ -1,30 +1,79 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Header from 'components/header'
 import { Container } from 'react-bootstrap'
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
 import Home from 'screens/Home'
 import About from 'screens/About'
 import Faq from 'screens/Faq'
 import ContactUs from 'screens/ContactUs'
+import Guide from 'screens/Guide'
 import { GlobalStyle, MainContainer } from 'assets/css/GlobalStyle'
+import ApiCall from 'api'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import { saveResponseData } from 'state/Response'
+
+const App = (props) => {
+  const { ppeGuides, ppeItems } = props
+
+  const getApiResponseData = () => {
+    ApiCall((response) => {
+      const { data } = response
+      props.updateResponseData(data)
+    })
+  }
+
+  if (!ppeGuides) {
+    getApiResponseData()
+  }
 
-const App = () => {
   return (
     <Container>
       <Router>
         <Header />
         <GlobalStyle />
         <MainContainer>
           <Switch>
-            <Route exact path="/" component={Home} />
+            <Route
+              exact
+              path="/"
+              component={() => (
+                <Home ppeGuides={ppeGuides || []} ppeItems={ppeItems || []} />
```

Comment:
I don't think the home screen should be needing this passed in. In the screen we should be selecting the data from Redux. I think Chris has previously given some advice on writing selectors?

We basically want to have in home screen something like:
```
const ppeGuides = useSelector(selectPPEGuides)
```

and in our state
```
export const selectPPEGuides = ({ state }) => state.ppeGuides.
```